internal/services/controllers/v1/olap/controller.go

```diff
@@ -180,33 +180,41 @@ func New(fs ...OLAPControllerOpt) (*OLAPControllerImpl, error) {
 	}
 
 	// Default jitter value
-	jitterMs := 1500
+	jitter := 1500 * time.Millisecond
 
 	// Override with config value if available
-	if o.olapConfig != nil && o.olapConfig.OpsJitter > 0 {
-		jitterMs = o.olapConfig.OpsJitter
+	if o.olapConfig != nil && o.olapConfig.Jitter > 0 {
+		jitter = time.Duration(o.olapConfig.Jitter) * time.Millisecond
+	}
+
+	// Default poll interval
+	pollInterval := 15 * time.Second
+
+	// Override with config value if available
+	if o.olapConfig != nil && o.olapConfig.PollInterval > 0 {
+		pollInterval = time.Second * time.Duration(o.olapConfig.PollInterval)
 	}
 
 	o.updateTaskStatusOperations = queueutils.NewOperationPool(
 		opts.l,
-		time.Second*15,
+		pollInterval,
```

Comment:
this now defaults to 5s to match task ops

Reply:
This isn't the polling interval, it's the context timeout. Polling is done with a scheduler on `Start` (I've been wanting to move this to the operation pool for a long time, but we're not there yet)